index.html

```diff
@@ -7,5 +7,42 @@
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header id="header">
+      <div id="owner_name">
+        <p>Derbal Amine</p>
+      </div>
+      <div id="header_menu">
+        <img src="images/icons8-menu-32.png" alt="header_menu" width="20px" />
+      </div>
+    </header>
+    <section id="headline">
+      <div id="introduction">
+        <p>Hey There.</p>
+        <p>I'm Amine</p>
```

Comment:
- [ ] Kindly I recommend you to use `h1` tag for "Main Title(Hey There I'm Amine)",  This way will be semantically correct and the screen readers and other technologies will thank you,[Check this link](https://www.reliablesoft.net/h1-tag/) for more info..